docker/ceph/set-start-env.sh

```diff
@@ -30,6 +30,19 @@ if [[ "$CEPH_DEBUG" == 1 ]]; then
     RGW_DEBUG='--debug-rgw=20 --debug-ms=1'
     VSTART_OPTIONS="$VSTART_OPTIONS -d"
 fi
+if [[ "$VSTART_CEPHADM" == 1 ]]; then
+    VSTART_OPTIONS+=" --cephadm"
+    if [[ ! -f "/root/.ssh/id_rsa" ]]; then
+        mkdir -p ~/.ssh
+        chmod 700 ~/.ssh
+        ssh-keygen -b 2048 -t rsa -f ~/.ssh/id_rsa -q -N ""
+    fi
+    if [[ ! -f "/root/.ssh/known_hosts" ]]; then
```

Comment:
id_rsa.pub should be inside `/root/.ssh/known_hosts` in order for asyncssh to work. It worked on a bare metal cluster with `vstart --cephadm`.

After that, there are some errors regarding the hostname and ip which I couldn't fix today.

Reply:
> id_rsa.pub should be inside `/root/.ssh/known_hosts` in order for asyncssh to work. It worked on a bare metal cluster with `vstart --cephadm`.
> 
> After that, there are some errors regarding the hostname and ip which I couldn't fix today.
Related to ssh?

Reply:
> id_rsa.pub should be inside `/root/.ssh/known_hosts` in order for asyncssh to work. It worked on a bare metal cluster with `vstart --cephadm`.

> After that, there are some errors regarding the hostname and ip which I couldn't fix today.

   **Related to ssh?**

Reply:
Yes, if you try `ssh root@localhost` it won't work because of that.

Reply:
Thanks @pereman2 !

Reply:
So far in this case the real failure was this:
```
debug1: trying public key file /root/.ssh/authorized_keys
debug1: Could not open authorized keys '/root/.ssh/authorized_keys': No such file or directory
```

Reply:
> id_rsa.pub should be inside `/root/.ssh/known_hosts` in order for asyncssh to work. It worked on a bare metal cluster with `vstart --cephadm`.

I just noticed I wanted to say `/root/.ssh/authorized_keys` instead of `known_hosts`

Reply:
If you run `cat ~/.ssh/id_rsa.pub >> /root/.ssh/authorized_keys` it should be fixed.